source/bootloader_common.cpp

```diff
@@ -17,6 +17,9 @@
 // ----------------------------------------------------------------------------
 
 #include "bootloader_common.h"
+#include "mbed.h"
```

Comment:
What is the reason to include `mbed.h`?

Reply:
Yes. Bootloader is an application.
Therefore I'm including `mbed.h` instad of specific header files.